### ISSUE TITLE
libxxhash for 1.1-maint

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#5400]
http://paste.debian.net/1166236/

**Comment by ThomasWaldmann** [comment]
```
    building 'borg.algorithms.checksums' extension
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/user/w/borg-env/include -I/home/user/.pyenv/versions/3.5.3/include/python3.5m -c src/borg/algorithms/checksums.c -o build/temp.linux-x86_64-3.5/src/borg/algorithms/checksums.o
    src/borg/algorithms/checksums.c:838:17: error: field ‘state’ has incomplete type
       XXH64_state_t state;
                     ^~~~~
```

**Comment by ThomasWaldmann** [comment]
when including the .c file, it works, strange.

**Comment by ThomasWaldmann** [comment]
issue is also seen on travis: https://travis-ci.org/github/borgbackup/borg/jobs/734155309 (before the .h -> .c change)

**Comment by ThomasWaldmann** [comment]
TODO: test with libxxhash installed on the system.

**Comment by LocutusOfBorg** [comment]
thanks, I was trying to figure out that exact issue today!

**Comment by ThomasWaldmann** [comment]
Note: i tested on misc. OSes and dists, with and without libxxhash on the system: looks good now.

**Comment by ThomasWaldmann** [comment]
fixed by PR #5404.

**Comment by LocutusOfBorg** [comment]
thanks again!
